src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -64,10 +64,10 @@ const MetaTraderUI = (() => {
     };
 
     const populateWebLinks = (server_info) => {
-        const $mt5_web = $('.mt5-web');
-        const $web_links = $mt5_web.find('a');
+        const mt5_url = `https://trade.mql5.com/trade${server_info && `?servers=${server_info.environment}&trade_server=${server_info.environment}`}`;
```

Comment:
Ah, to clarify my intention here:
```
 const mt5_url = `https://trade.mql5.com/trade`;
```
would go at the top of the file, along with any other immutable constants, so that they're obvious when you look at it from the start.

Just a suggestion / coding-style, though, so feel free to ignore.

Reply:
Oh yes, this is a good suggestion. I splitted the domain part of URL from query string and moved it on top of the file. 👍🏻